app/index.js

```diff
@@ -793,11 +793,11 @@ ipcMain.on("create-session", (_, name, adblock, cache) => {
         }
         const filename = item.getFilename()
         let save = joinPath(downloadSettings.downloadpath, filename)
-        let duplicateNumber = 1
+        let duplicateNumber = 0
         let newFilename = item.getFilename()
         while (isFile(save)) {
             duplicateNumber += 1
-            const extStart = filename.lastIndexOf(".")
+            const extStart = filename.indexOf(".")
```

Comment:
This does not always have the right effect either, for example:
![image](https://user-images.githubusercontent.com/1696674/161091708-b9f761d9-1c5e-4d9d-b691-07f9a30a5857.png)
![image](https://user-images.githubusercontent.com/1696674/161092654-c3ea956d-83ff-4129-bae9-a271e9296a27.png)

I propose that we add an exception for `tar.something` files since those are probably the only cases where `lastIndexOf` is not working right currently. I'm not a fan of using a regular `indexOf`, because when there are dots in the filename it's going to be messed up. What are your thoughts?